packages/mermaid/src/mermaidAPI.ts

```diff
@@ -101,19 +101,16 @@ export interface RenderResult {
 
 async function parse(text: string, parseOptions?: ParseOptions): Promise<boolean | void> {
```

Comment:
Thanks for the fix!

As far as I'm aware, this function now always returns `true` or `false`, so we might as well get rid of the `void` here.

(plus, that way TypeScript will throw an error if `#1066` happens again!)

```suggestion
async function parse(text: string, parseOptions?: ParseOptions): Promise<boolean> {
```

You may want to change this locally, instead of using GitHub's UI, since I think you'll need to regenerate `docs/config/setup/modules/mermaidAPI.md`.